src/main/java/com/toofifty/goaltracker/ui/TaskItemContent.java

```diff
@@ -42,7 +42,10 @@ public void refresh()
     {
         titleLabel.setText(task.toString());
         titleLabel.setForeground(STATUS_TO_COLOR.get(task.getStatus()));
+
+        int indent = 16 * task.getSubtaskRank();
         iconLabel.setIcon(iconService.get(task));
+        iconLabel.setBorder(new EmptyBorder(4, indent, 0, 0));
```

Comment:
Mind removing the top padding from this?

```suggestion
        iconLabel.setBorder(new EmptyBorder(0, indent, 0, 0));
```